### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by LW95x**
Not really sure how to structure this, or how detailed to be with it either. 

Also not to sure how to format a readme file so it looks cleaner, i.e. with headers.

 Any tips to further clarify would be appreciated, thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Northcoders News API
 
-You will need to create two .env files: .env.test and .env.development. 
+Hosted version: https://be-nc-news-sfsv.onrender.com
```
Comment (nc-jonathan-mcguire) [inline README.md]: i would change this to a link to an actual working route i.e./api?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+
+- An API that has been designed to mimic the building of a real world backend service (such as Reddit), to provide information to front end architecture.
+
+Cloning/dependencies/seeding/testing
+
+To clone - 'git clone https://github.com/LW95x/be-nc-news.git'
+
+Installing dependencies - see 'package.json' for list of dev dependencies not included with cloning. You will also need to run 'npm install'.
+
+Seeding - Not sure what to put here?
```
Comment (nc-jonathan-mcguire) [inline README.md]: Maybe the command to seed the database? Presume theres a script to do in the package.json

**Review by nc-jonathan-mcguire — CHANGES_REQUESTED** [review]
Just two bits ive put in the comments

**Review by nc-mick-fay — APPROVED** [review]
Good changes to address previous comments - merge into main when you're ready

**Pull request merged**